docs/DeveloperGuide.md

```diff
@@ -308,18 +313,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Update Contact Information**
+
+**MSS**
+
+1.  User requests to list TAs
+2.  AddressBook shows a list of TAs
+3.  User requests to update the contact of a specific TA in the list and key in the necessary information
+4.  AddressBook updates the contact information of that TA
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+* 3b. The given contact information is invalid.
+  * 3b1. AddressBook shows an error message.
+    
+    Use case resumes at step 2.
+
+**Use case: View TAs of specific course**
+
+**MSS**
+
+1.  User requests to list courses
+2.  AddressBook shows a list of courses
+3.  User requests to show TAs of a specific course
+4.  AddressBook shows a list of TAs of specific course
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given course is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+
 *{More to be added}*
 
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4.  Data should persist across user sessions
+5.  Project should be able to handle information from across academic years
 
 *{More to be added}*
 
 ### Glossary
 
+* **Course**: A program students are enrolled in to work towards a degree
+* **Teaching Assistant**: Students who support the teaching of a course
```

Comment:
Maybe we should clarify here that this is what TA stands for.

Reply:
resolved in [c7f1d00](https://github.com/AY2324S1-CS2103T-T10-1/tp/pull/9/commits/c7f1d00e7e2af5f91627cdf126c76fc7c115e335)